packages/validation/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "@trezor/validation",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "type": "module",
+    "main": "src/index",
+    "scripts": {
+        "lint": "eslint '**/*.{ts,tsx,js}'",
+        "test:unit": "jest",
```

Comment:
please delete this tests script + `jest.config.js` since it is unused, we can easily bring it back if we will need it

Reply:
done